packages/wrapped-keys/src/lib/constants.ts

```diff
@@ -1,10 +1,10 @@
 import { Network, KeyType } from './types';
 
-export const CHAIN_ETHEREUM = 'ethereum';
-export const LIT_PREFIX = 'lit_';
+export const CHAIN_ETHEREUM = 'ethereum' as const;
+export const LIT_PREFIX = 'lit_' as const;
 
-export const NETWORK_EVM: Network = 'evm';
-export const NETWORK_SOLANA: Network = 'solana';
+export const NETWORK_EVM: Network = 'evm' as const;
+export const NETWORK_SOLANA: Network = 'solana' as const;
 
```

Comment:
What's the benefits of asserting a string a literal type? eg. the type `CHAIN_ETHEREUM` has to be exactly `ethereum`? does it help with passing this const value as a key to other constant values?

Reply:
so the TS type of NETWORK_EVM as Network is `string`, because Network is a union of `string`s... but the TS type of `NETWORK_EVM` as Network `as const`, is a string _literal_ of `evm`.  `as const` is basically telling TS 'Use the narrowest possible type for this thing'.  It matters when you try to compose objects using these strings or use these strings as indexes into object hashes that use them as keys.